apier/v1/suppliers_it_test.go

```diff
@@ -971,6 +972,60 @@ func testV1SplSRemSupplierProfiles(t *testing.T) {
 	}
 }
 
+func testV1SplSSetSupplierProfilesWithoutRatingPlanIDs(t *testing.T) {
```

Comment:
Please move this test into general_tests since in the apier we should keep structure simple